src/actions/index.js

```diff
@@ -12,26 +12,28 @@ export function fetchSearchIndexSuccess(searchIndex) {
     type: actionTypes.FETCH_SEARCH_INDEX_SUCCESS,
     searchIndex,
   };
-}
+} 
 
 export function fetchSearchIndexFailure() {
   return {
     type: actionTypes.FETCH_SEARCH_INDEX_FAILURE,
   };
 }
 
-export const getSearchResultsRequest = searchInput => ({
```

Comment:
Not sure why you deleted these - we still need them.

Reply:
I accidentally deleted these lines.

Reply:
I merged the changes from master onto this branch and that resolved this issue